src/complete.js

```diff
@@ -0,0 +1,12 @@
+import { store, list } from './index.js';
+
+const updateList = (lists, check) => {
+  const listInLists = list.find((t) => t.description === lists.description);
+
+  listInLists.completed = check;
+
+  lists.completed = check;
+  store();
+};
+
+export default updateList;
```

Comment:
- [ ] this function is created but never used anywhere in the project, please import it to `index.js` and use it
- [ ] it should update the item's status when used, you can make an event listener whenever the check input is clicked